libcxx/test/std/experimental/simd/simd.class/simd_copy.pass.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 // UNSUPPORTED: c++03, c++11, c++14
+// UNSUPPORTED: hwasan
```

Comment:
I think it would be better to instead use `REQUIRES: long_tests` since this isn't a fundamental limitation of `hwasan` but merely a matter of timing out on your specific builder (I assume?).